TODO.md

```diff
@@ -1,9 +1,17 @@
 ## TODOs before merging to `future`
 
-- support for RR with unwrap for commutative commands
-- working on all tests
+- fix tests from compiler/test_evaluation_scripts.sh:
+  + bigrams
 - clean up utils for annotations
 - graphviz
 - Changing PaSh flags (making the default be priority r-split and then consecutive chunks), so remove the r_split flag and make defaults be the ones from the OSDI paper
 - Fixing annotation library installation to a specific commit
-- Remove code which got obsolete due to the changes
\ No newline at end of file
+- Remove code which got obsolete due to the changes
+- Room for optimization: basically disable parallelization after a tr which squeezes all new lines since there are no sequences of data to parallelize anyway for the moment. 
+    Long-term, we could allow parallelization but with a adj_line_merge aggregator.
+- Changes to scripts:
+  + `spell-grep.sh`: added `export -f set_diff` as the command was not known when parallelizing?
```

Comment:
Could this export be removed if it was not needed in the end? @tammam1998 did we actually need to export all functions for `r_wrap` to work?

Reply:
Oh, this was an inconsistency with the TODOs. I had removed the `export` already.

Reply:
Great, feel free to squash and merge if you are done!